go.mod

```diff
@@ -15,7 +16,7 @@ require (
 	github.com/operator-framework/operator-lib v0.11.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.63.0
 	github.com/prometheus/client_golang v1.14.0
-	github.com/prometheus/client_model v0.3.0
```

Comment:
Seems like because of this change, you need to run `go mod vendor` that will generate new content in `vendor` folder, then you need commit it to avoid failure of `check-commited-changes` job.